src/usecase_clustering.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+
+# Clustering
+
+```{r echo = FALSE}
+set.seed(1234)
+```
+
+This is a use case for clustering with the [%mlr] package. We consider the [agriculture](https://www.rdocumentation.org/packages/cluster/versions/1.10.0/topics/agriculture) dataset that contains observations about $n=12$ countries including  
+
+* the GNP (Gross National Product) per head (\texttt{x}) ,
+* the percentage in agriculture (\texttt{y}).
+
+So let's have a look at the data first.
+
+```{r, fig.width = 5}
+library("cluster")
+data(agriculture)
+
+plot(y ~ x, data = agriculture)
+```
+
+
+We aim to group the observations into clusters that contain similar objects. We will 
+
+* define the learning task  ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/task/index.html)),
+* select a learning method ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/learner/index.html)),
+* train the learner with data ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/train/index.html)), 
+* evaluate the performance of the model ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/performance/index.html)) and
+* tune the model ([here](http://mlr-org.github.io/mlr-tutorial/devel/html/tune/index.html)).
+
+### Defining a task
+
+We now have to define a clustering task. Notice that a clustering task doesn't have a target variable. 
+
+```{r message = FALSE}
+library(mlr)
+agri.task = makeClusterTask(data = agriculture)
+agri.task
+```
+
+Calling the task again shows us some basic informations as the number of observations, the data types of the features or if there are still some missing values, that should have been preprocessed.
+
+### Defining a learner
+
+We generate the learner by calling ``makeLearner`` and specifying the learning method, and, if needed, hyperparameters.
+
+An overview over all learners can be found  [here](http://mlr-org.github.io/mlr-tutorial/devel/html/integrated_learners/index.html). You can also call the \texttt{listLearners} command for our specific task.
```

Comment:
* Please change the link to `[here](integrated_learners.md)`
* Please use also a link to `[&listLearners]`